tests/integration/cmor/_fixes/cmip6/test_mcm_ua_1_0.py

```diff
@@ -157,3 +158,61 @@ def test_tas_fix_metadata(cubes):
     assert out_cubes_2[0].var_name == 'tas'
     coord = out_cubes_2[0].coord('height')
     assert coord == height_coord
+
+
+@pytest.fixture
+def thetao_cubes():
+    time_coord = iris.coords.DimCoord(
+        [0.0004, 1.09776], var_name='time', standard_name='time',
+        units='days since 1850-01-01 00:00:00')
+    lat_coord = iris.coords.DimCoord(
+        [0.0, 1.0], var_name='lat', standard_name='latitude', units='degrees')
+    lon_coord = iris.coords.DimCoord([-0.9375, 357.1875],
+                                     bounds=[[-1.875, 0.], [356.25, 358.125]],
+                                     var_name='lon',
+                                     standard_name='longitude')
+    lev_coord = iris.coords.DimCoord(
+        [5.0, 10.0], bounds=[[2.5, 7.5], [7.5, 12.5]],
+        var_name='lev', standard_name=None, units='m',
+        attributes={'positive': 'up'})
+    coord_specs = [
+        (time_coord, 0),
+        (lev_coord, 1),
+        (lat_coord, 2),
+        (lon_coord, 3),
+    ]
+    thetao_cube = iris.cube.Cube(
+        np.ones((2, 2, 2, 2)),
+        var_name='thetao',
+        dim_coords_and_dims=coord_specs,
+    )
+
+    return iris.cube.CubeList([thetao_cube])
+
+
+def test_get_thetao_fix():
+    """Test getting of fix."""
+    fix = Fix.get_fixes('CMIP6', 'MCM-UA-1-0', 'Omon', 'thetao')
+    assert fix == [AllVars(None)]
+
+
+def test_thetao_fix_metadata(thetao_cubes):
+    """Test ``fix_metadata`` for ``thetao``."""
+    fix = AllVars(None)
+    out_cubes = fix.fix_metadata(thetao_cubes)
+    assert out_cubes is thetao_cubes
+    assert len(out_cubes) == 1
+    out_cube = out_cubes[0]
+
+    # Check data of longitude
+    lon_coord = out_cube.coord('longitude')
+    assert lon_coord.points[0] == 0.9375
+    assert np.all(lon_coord.bounds[0] == [0., 1.875])
```

Comment:
I think that test is wrong here. If you roll the data by `-1` (i.e. you shift all values to the right by 1) and do the same to the `longitude` (plus adding 360 to all negative values), you would end up with `[357.1875, 359.0625]` (starting from `[-0.9375, 357.1875]`]). Similarly, the bounds should read `[[356.25, 358.125], [358.125, 360.0]]`.

For testing float arrays, you should always use `np.testing.assert_allclose`:

```suggestion
    np.testing.assert_allclose(lon_coord.points, [357.1875, 359.0625])
    np.testing.assert_allclose(lon_coord.bounds,
                               [[356.25, 358.125], [358.125, 360.0]])
```